MyCryptoHQ/MyCrypto
# Hide refresh button and its spinner on initial balance fetch.

Closes #1070

### Description

Hides refresh button & its spinner when there's no balance (e.g. initial fetch)

### Steps to Test

1. Set network to "Slow 3g" or just have slow internet
2. Open a wallet
3. Observe one spinner on your balance
4. When it loads, refresh the balance
5. Observe still one spinner next to your balance
